domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -452,7 +457,13 @@ public SampleClaim withReDeterminationRequestedAt(LocalDateTime reDeterminationR
     }
 
     public SampleClaim withSealedClaimDocument(URI sealedClaimDocument) {
-        this.sealedClaimDocument = sealedClaimDocument;
+        ClaimDocument claimDocument = new ClaimDocument(sealedClaimDocument,
+            "001CLAIM-FORM",
+            ClaimDocumentType.SEALED_CLAIM,
+            LocalDateTimeFactory.nowInLocalZone(),
+            LocalDateTimeFactory.nowInLocalZone(),
+            null);
```

Comment:
this null doesn't explain what is purpose for this, can be extracted to variable or property with meaningful name